[INF-535] Fix web-test

sliptype opened the pull request:
### Description

Was able to get tests to not hang by removing the mock for `remote-config-instance` in cache/store.test.ts

The tests pass without out so not sure if/why it was needed.

Also added `threads: false` to the vitest config and excluded `dist` which was causing issues for me locally

### How Has This Been Tested?

![image](https://github.com/AudiusProject/audius-protocol/assets/19916043/8e7db55a-9d1d-45e8-a4e4-8f97dd70aae5)

rickyrombo reviewed (approved):
I just realized that when we reverted vitest back we didn't change the config - the "old" config is "threads: false" or "num threads: 1" but the "new" config is "numWorkers" , hence why it went back to mulithreaded when we reverted!

Excited for this find! I'm surprised the test passed with the mock deleted - also surprised the mock broke it at all.. I wonder if threads: false was the true fix?

sliptype commented:
> I just realized that when we reverted vitest back we didn't change the config - the "old" config is "threads: false" or "num threads: 1" but the "new" config is "numWorkers" , hence why it went back to mulithreaded when we reverted!
> 
> Excited for this find! I'm surprised the test passed with the mock deleted - also surprised the mock broke it at all.. I wonder if threads: false was the true fix?

Even with `threads: false` the tests were hanging, I had to bisect the tests and narrowed it down to that one file. Then commenting out the mock caused the tests to not hang and everything passed 🤷

audius-infra commented:
Preview this change https://demo.audius.co/sk-fix-web-test
